src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
+    public static final int BALLS_COUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[BALLS_COUNT];
+        for (int i = 0; i < balls.length; i++) {
```

Comment:
since we already have a constant I'd use it instead
```suggestion
        for (int i = 0; i < BALLS_COUNT; i++) {
```